restli-int-test-server/src/main/java/com/linkedin/restli/examples/greetings/server/GreetingsResourceImpl.java

```diff
@@ -200,6 +200,10 @@ public CreateResponse create(Greeting entity, @QueryParam("isNullId") @Optional(
   @RestMethod.Get
   public Greeting get(Long key)
   {
+    if (key == 204L)
+    {
+      throw new RestLiServiceException(HttpStatus.S_204_NO_CONTENT);
```

Comment:
Can you add description for this PR? like explaining what real problem this PR is fixing?
Customers throw RestLiServiceException for httpStatus 204? We want to catch such exception and return null for response content?

Reply:
I think I mentioned it in the title. I will provide the empty description a description

Reply:
Description added